### PR TITLE
Dev/cmcl 90 physical camera support

### PR DESCRIPTION
**Pull request by glabute**
Add support for expressing FOV as horizontal or vertical, according to the current setting in the driven Camera.
This turned out to be rather tricky, and required a refactor of the way the lens is drawn (can no longer be a PropertyDrawer because it needs to access the Camera).

image

Also addressed in this PR: a fix for fogbugz 1060230, corrected by snapshotting the appropriate values form the Camera when drawing the inspector (and not waiting for RoundRobin to update them).

And another bonus: when the Lens setting foldout is collapsed, we still allow the FOV to be edited (it's the most common usecase).

image

### CODE REVIEW

File: Editor/Editors/CinemachineVirtualCameraBaseEditor.cs [hunk 1]
```diff
@@ -232,20 +247,349 @@ protected void DrawGlobalControlsInInspector()
                 = EditorGUILayout.Toggle(
                     new GUIContent(
                         "Game Window Guides",
-                        "Enable the display of overlays in the Game window.  You can adjust colours and opacity in Edit/Preferences/Cinemachine."),
+                        "Enable the display of overlays in the Game window.  "
+                            + "You can adjust colours and opacity in Edit/Preferences/Cinemachine."),
```
Comment (gaborkb) [inline Editor/Editors/CinemachineVirtualCameraBaseEditor.cs]: I could not find anything under Edit. I don't have Preferences there.

File: Editor/Editors/CinemachineVirtualCameraBaseEditor.cs [hunk 2]
```diff
@@ -232,20 +247,349 @@ protected void DrawGlobalControlsInInspector()
...
+                    EditorGUILayout.PropertyField(property.FindPropertyRelative(() => m_LensSettingsDef.LensShift));
+#endif
+                }
+                EditorGUILayout.PropertyField(property.FindPropertyRelative(() => m_LensSettingsDef.Dutch));
+                --EditorGUI.indentLevel;
+            }
+            property.serializedObject.ApplyModifiedProperties();
+        }
+
+        static float ExtraSpaceHackWTF() { return EditorGUI.indentLevel * (EditorGUIUtility.singleLineHeight - 3); }
```
Comment (gaborkb) [inline Editor/Editors/CinemachineVirtualCameraBaseEditor.cs:495]: :D 👍

**Review by gaborkb — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -20,8 +20,8 @@
             "path": "Samples~/Cinemachine Example Scenes"
         },
         {
-            "displayName": "Scrub Bubble support for Nested Timelines (experimental)",
-            "description": "Experimental feature.  Adds a Timeline extension to support Scrubb Bubble for nested timelines.  This extension will become obsolete when Timeline adds native support for the missing API",
+            "displayName": "Cached Scrubbing support for Nested Timelines (experimental)",
+            "description": "Experimental feature.  Adds a Timeline extension to support Cached Scrubbing for nested timelines.  This extension will become obsolete when Timeline adds native support for the missing API",
             "path": "Samples~/Nested Timeline Scrub Bubble"
```
Comment (gaborkb) [inline package.json:25]: Did a search for Scrub Bubble, and found in CinemachineTimelineEditorExtensions.cs. 
And the name of the samples is still scrub bubble here.

**Review by gaborkb — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (glabute) [reply package.json:25]: It's not a dirty word... I think we can leave it

**Review by gaborkb — APPROVED** [review]

**Pull request merged**